core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -213,7 +214,7 @@ public function fromUuid(string $uuid): self
     {
         $this->lastException = null;
 
-        $filesModel = $this->filesModelAdapter()->findByUuid($uuid);
+        $filesModel = $this->getFilesModelAdapter()->findByUuid($uuid);
```

Comment:
In case you wondered: this naming scheme was taken from Symfony. :slightly_smiling_face: 
(https://symfony.com/doc/current/service_container/service_subscribers_locators.html#service-subscriber-trait)